cart-service/src/validators/env.validators.ts

```diff
@@ -48,9 +49,9 @@ const envValidators = [
     referencedBy: 'environmentVariables',
   }),
 
-  standardString(['connectorBasicAuthPassword'],{
-    code: 'connectorBasicAuthPassword',
-    message: 'Basic auth password should be a valid string',
+  longString(['basicAuthPwdCurrent'],{
```

Comment:
Might be better to use the `standardString` function and pass in the min and max values using `overrideConfig` like `clientSecret` does?

Reply:
updated